packages/dev/docs/src/Layout.js

```diff
@@ -116,7 +117,8 @@ function Page({children, currentPage, publicUrl, styles, scripts}) {
       dir="ltr"
       prefix="og: http://ogp.me/ns#"
       className={clsx(
-        theme.global.spectrum,
+        pageStyles.spectrum,
+        Object.values(theme.global),
```

Comment:
what happened here?

Reply:
Some of the classes we apply in provider didn't seem to be applied here, which resulted in some broken styles. Honestly not sure how it was working before. 😂

Reply:
Taking a look at the diffs, `<kbd>` didn't get the `keyboard` class:
Old:
<img width="168" alt="Old" src="https://user-images.githubusercontent.com/8961049/146843312-7db0be32-f52d-4e43-9dfb-7e9ff8477d76.png">
New:
<img width="168" alt="New" src="https://user-images.githubusercontent.com/8961049/146843356-29a67996-a645-4358-8d53-a698e0791f56.png">